src/pugixml.cpp

```diff
@@ -78,6 +78,9 @@
 #	pragma option push
 #	pragma warn -8008 // condition is always false
 #	pragma warn -8066 // unreachable code
+#	define PUGI_IMPL_BORLANDC_CONST_CAST(to, from) const_cast<to>(from)
+#else
+#	define PUGI_IMPL_BORLANDC_CONST_CAST(to, from) from
```

Comment:
Missing #undef at the bottom of the file. (I need to separately look at whether we even still need this because this workaround is probably 15 years old now but for now it's fine to have it wrapped in a macro)

Reply:
can I just remove it instead? That would keep the code simpler and avoid a compat macro

Reply:
Yeah that would work